app/controllers/import_controller.rb

```diff
@@ -38,108 +46,140 @@ def import
       # This is a catch all for any other unexpected error
-      @errors << "Unexpected Error: '#{e&.message}' Please make sure that .xlsx import file is formatted in accordance with the formatting guidance."
+      @errors << "Unexpected Error: '#{e&.message}' Please make sure that .#{format == :epix ? 'csv' : 'xlsx'} import file is formatted in accordance with the"\
+                 ' formatting guidance.'
     end
 
     render json: { patients: @patients, errors: @errors }
   end
 
+  private
+
+  def import_sara_alert_format_field(patient, field, row, row_ind, col_num, workflow, valid_jurisdiction_ids)
+    if field == :jurisdiction_path
+      patient[:jurisdiction_id], patient[:jurisdiction_path] = validate_jurisdiction(row[SARA_ALERT_FORMAT_FIELDS.index(:jurisdiction_path)],
+                                                                                     row_ind, valid_jurisdiction_ids)
+    elsif field == :assigned_user
+      patient[:assigned_user] = import_assigned_user(row[SARA_ALERT_FORMAT_FIELDS.index(:assigned_user)])
+    elsif field == :symptom_onset && workflow == :isolation
+      patient[:user_defined_symptom_onset] = row[SARA_ALERT_FORMAT_FIELDS.index(:symptom_onset)].present?
+      patient[field] = import_field(field, row[col_num], row_ind)
+    # TODO: when workflow specific case status validation re-enabled: uncomment
+    # elsif field == :case_status
+    #   patient[field] = validate_workflow_specific_enums(workflow, field, row[col_num], row_ind)
+    else
+      # TODO: when workflow specific case status validation re-enabled: this line can be updated to not have to check the case_status field
+      patient[field] = import_field(field, row[col_num], row_ind) unless %i[symptom_onset case_status].include?(field) && workflow != :isolation
+    end
+  end
+
+  def import_epi_x_field(patient, field, row, row_ind, col_num, international_address)
+    return if field == :foreign_address_country && !international_address
+
+    if %i[travel_related_notes port_of_entry_into_usa].include?(field) # fields represented by multiple columns
+      if patient[field].blank?
+        patient[field] = "#{EPI_X_HEADERS[col_num]}: #{import_field(field, row[col_num], row_ind)}"
+      else
+        patient[field] += ", #{EPI_X_HEADERS[col_num]}: #{import_field(field, row[col_num], row_ind)}"
+      end
+    elsif field == :secondary_telephone && row[EPI_X_FIELDS.index(:primary_telephone)].blank? # populate primary telephone before secondary
+      patient[:primary_telephone] = import_field(field, row[col_num], row_ind)
+    elsif international_address && %i[address_line_1 address_city address_state address_zip address_line_2].include?(field)
+      patient[FOREIGN_ADDRESS_MAPPINGS[field]] = import_field(field, row[col_num], row_ind)
+    elsif %i[date_of_birth date_of_departure].include?(field)
+      patient[field] = import_field(field, Date.strptime(row[col_num], '%m/%d/%Y'), row_ind) if row[col_num].present?
```

Comment:
So I realized parsing dates like this opens up a new vector for getting 0 dates into the system. If someone incorrectly uses a two-digit year on a date, e.g. 6/9/00 (instead of 6/9/2000), the date will successfully save with year 0.

I'm thinking we need to add another check (on top of what was done in #912) in the `DateValidator` for day, month, or year equal to 0 in order protect against this.

Reply:
It is hitting the `DateValidator` even with that commented out though.

I think what's happening is our `DateValidator` overrides the default `DateValidator`, which runs on certain fields as a result of this:
https://github.com/SaraAlert/SaraAlert/blob/1368-epix/app/models/patient.rb#L60-L68

Reply:
Addressed here https://github.com/SaraAlert/SaraAlert/pull/884/commits/649adfa78fd49a66a4abc20a979d803f6405e736